src/index.html

```diff
@@ -4,19 +4,104 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Moyo Header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="container">
+      <header class="header">
+        <a
+          href="#"
+          class="img"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="items">
+            <li>
+              <a
+                href="#"
+                class="item is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li>
+              <a
+                href="#"
+                class="item"
+              >
+                Samsung
+              </a>
+            </li>
+            <li>
+              <a
+                href="#"
+                class="item"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li>
+              <a
+                href="#"
+                class="item"
+                data-qa="hover"
```

Comment:
The `href` attribute in the anchor tag should not contain a space. If this is meant to be a placeholder link, use `#` without any spaces.